ddtrace/appsec/_metrics.py

```diff
@@ -12,6 +12,8 @@
 
 DDWAF_VERSION = ddwaf.version()
 
+bool_str = ("false", "true")
```

Comment:
can we re-use from ddtrace.internal.utils.formats import asbool istead of a new validator?

Reply:
it's not a validator, more a simple format writer. This is faster than str(b).lower()